timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/LocalPaxosComponents.java

```diff
@@ -194,8 +195,10 @@ private Components createComponents(Client client) {
         // TODO (jkong): This test is no longer valid with the new implementation, it needs to be fixed before
         // the migration.
         if (!canCreateNewClients && clientDirectoryDoesNotExist(legacyClientDir)) {
-            throw new ServiceNotAvailableException("This TimeLock server is not allowed to create new clients at this"
-                    + " time, and the client " + client + " provided is novel for this TimeLock server.");
+            throw new ServiceNotAvailableException(
+                    "This TimeLock server is not allowed to create new clients at this time, and the"
+                            + " client {} provided is novel for this TimeLock server.",
+                    UnsafeArg.of("client", client));
```

Comment:
Based on other usage, I think we've decided elsewhere that this is safe. It's basically the namespace with which clients call timelock.